### PR TITLE
Update solc

### PR DESCRIPTION
**Pull request by svc-ghb-is-04**
- [x] Review the [latest Solidity changes](https://github.com/ethereum/solidity/blob/develop/Changelog.md) and add test files for any new language constructs.
- [x] If tests are failing: fix the schema and update the changelog accordingly.

### CODE REVIEW

**Comment by frangio** [comment]
The new require for custom errors doesn't need any AST changes.

File: test/sources/errors-require.sol [hunk 1]
```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.8.26;
+
+/// @dev docs
+error InsufficientBalance(uint available, uint required);
+
+contract Foo {
+    error OhNo();
```
Comment (ericglau) [inline test/sources/errors-require.sol:9]: This error isn't used

**Review by ericglau — APPROVED** [review]
LGTM, just one minor comment

File: test/sources/errors-require.sol (hunk 1 above)
Comment (frangio) [reply test/sources/errors-require.sol]: Oh No

**Pull request merged**